### ISSUE TITLE
Convert schema names with `-` to camel case, and return a warning saying this was done

### ISSUE THREAD

**Issue by sooraj1002** [techsavvyash/dynamo-prisma#6]

**Comment by Himasnhu-AT** [comment]
We can use this codeblock

```
if (fieldData.includes('-')) {
     console.error()
     process.exit(1)
} else {
     // normal execution
}
```

**Comment by sooraj1002** [comment]
@Himasnhu-AT instead of throwing an error, lets format the schema name to camel case, and return a warning from the function.

**Comment by sooraj1002** [comment]
we need to extend this to schema names as well, not just field names

**Comment by techsavvyash** [comment]
this has been fixed with #27 , instead of using camel case we are using snake case and replacing all instances of `-` and ` ` with `_` and this happens at the schema level itself, we can send back the updated schema.
